cmd/main.go

```diff
@@ -178,6 +178,7 @@ func InitLogs() *logrus.Entry {
 	if err != nil {
 		logger.Error("Invalid Log Level: ", Options.LogConfig.LogLevel)
 	} else {
+		logger.Level = logLevel
 		log.SetLevel(logLevel)
```

Comment:
@omertuc do you think this line is still relevant or shall we just get rid of it as we would mostly use the instatiated logger?